clients/admin-ui/src/features/privacy-experience/form/helpers.tsx

```diff
@@ -51,22 +52,66 @@ const privacyCenterValidationSchema = Yup.object()
   .shape({
     title: Yup.string().required().label("Title"),
     description: Yup.string().required().label("Description"),
+    privacy_policy_link_label: Yup.string()
+      .optional()
+      .label("Privacy policy link label"),
+    privacy_policy_url: Yup.string()
+      .optional()
+      .trim()
+      .url()
+      .label("Privacy policy URL"),
   })
   .concat(buttonGroupValidationSchema);
 
 const bannerValidationSchema = Yup.object()
   .shape({
-    title: Yup.string().required().label("Banner title"),
-    description: Yup.string().required().label("Banner description"),
+    title: Yup.string().required().label("Overlay title"),
+    description: Yup.string().required().label("Overlay description"),
     acknowledge_button_label: Yup.string()
       .required()
       .label("Acknowledge button label"),
     privacy_preferences_link_label: Yup.string()
       .required()
       .label("Privacy preferences link label"),
+    banner_title: Yup.string().optional().label("Banner title"),
+    banner_description: Yup.string().optional().label("Banner description"),
+    privacy_policy_link_label: Yup.string()
+      .optional()
+      .label("Privacy policy link label"),
+    privacy_policy_url: Yup.string()
+      .optional()
+      .trim()
+      .url()
+      .label("Privacy policy URL"),
   })
   .concat(buttonGroupValidationSchema);
 
+// Coerce empty strings to "null" values for the update API
+// NOTE: this is identical to the transform for the create API below, but copied
+// into a separate function for Typescript type-safety
```

Comment:
nb: this little business-logic helper that coerces `""` to `null` is duplicated for both mutations. I didn't feel up to figuring out the Typescript incantation to support both type signatures in a single function.

Reply:
I think it'd be something like

```typescript
export const transformFormValuesToExperienceConfig = <T extends {banner_title: string ...}>(values: T) : T => ({
  ...
});

// or
// <T extends Pick<ExperienceConfigUpdateParams, 'banner_title' | 'banner_description'...>>
```

Reply:
Thanks... I'm going to keep it as is because I find that really tricky Typescript magic can be self-defeating when it comes to maintaining!